src/sentry/features/__init__.py

```diff
@@ -147,6 +147,8 @@
 default_manager.add("organizations:widget-library", OrganizationFeature, True)
 default_manager.add("organizations:widget-viewer-modal", OrganizationFeature, True)
 default_manager.add("organizations:widget-viewer-modal-minimap", OrganizationFeature, True)
+default_manager.add("organizations:dashboard-custom-measurement-widgets", OrganizationFeature, True)
```

Comment:
Is this feature flag (Line 150) meant to be here?

Reply:
hm i dont think so, ill get rid of it ty!